plugins/pubIds/urn/locale/nl_NL/locale.xml

```diff
@@ -12,65 +12,42 @@
-	<message key="plugins.pubIds.urn.manager.settings.URNsForJournalContent">Kies de objecten waar de URN's aan toegekend moeten worden.</message>
-	<message key="plugins.pubIds.urn.manager.settings.enableIssueURN">URN's worden toegekend aan nummers.</message>
-	<message key="plugins.pubIds.urn.manager.settings.enableArticleURN">URN's worden toegekend aan artikelen.</message>
-	<message key="plugins.pubIds.urn.manager.settings.enableGalleyURN">URN's worden toegekend aan proeven (bv. HTML of PDF bestanden).</message>
-	<message key="plugins.pubIds.urn.manager.settings.enableSuppFileURN">URN's worden toegekend aan aanvullende bestanden.</message>
-
-	<!-- custom URN suffix for an object -->
-	<message key="plugins.pubIds.urn.metadata">URN Metadata</message>
-	<message key="plugins.pubIds.urn.urnSuffix">URN achtervoegsel</message>
-	<message key="plugins.pubIds.urn.urnSuffix.description">Aangepast URN achtervoegsel voor dit artikel.</message>
-	<message key="plugins.pubIds.urn.calculateCheckNo">Bereken controlegetal</message>
-
-	<!-- consider URNs at other places in the system -->
-	<message key="plugins.pubIds.urn.rt.metadata.urn">Uniform Resource Name</message>
-
-	<!-- errors -->
-	<message key="plugins.pubIds.urn.manager.settings.form.journalContentRequired">Kies de objecten waaraan URN's toegekend moet worden.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnPrefixRequired">Geef een URN voorvoegsel.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnPrefixPattern"><![CDATA[Het URN voorvoegsel heeft de volgende vorm "urn:"&lt;NID&gt;":"&lt;NSS&gt;.]]></message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnIssueSuffixPatternRequired">Geef het URN achtervoegsel patroon voor nummers.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnArticleSuffixPatternRequired">Geef het URN achtervoegsel patroon voor artikelen.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnGalleySuffixPatternRequired">Geef het URN achtervoegsel patroon voor proeven.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnSuppFileSuffixPatternRequired">Geef het URN achtervoegsel patroon voor aanvullende bestanden.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.namespaceRequired">Kies een namespace.</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign">Wis URN's</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign.description">Als u de URN configuratie verandert heeft dat geen gevolgen voor reeds toegekende URN's. Nadat de gewijzigde URN configuratie is opgeslagen kunt u deze knop gebruiken om alle bestaande URN's te wissen om de nieuwe instellingen ook toe te passen op bestaande objecten.</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign.confirm">Weet u zeker dat u allle bestaande URN's wilt verwijderen?</message>
```

Comment:
this probably is a typo, s/allle/alle/, but I see that it has been in the file before :)